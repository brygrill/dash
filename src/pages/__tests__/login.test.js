import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import LoginPage from '../login';

describe('<LoginPage />', () => {
  it('Renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<LoginPage />, div);
  });
  it('Matches snapshot', () => {
    const component = renderer.create(<LoginPage />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
