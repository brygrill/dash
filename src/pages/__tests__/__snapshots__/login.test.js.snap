exports[`<LoginPage /> Matches snapshot 1`] = `
<div>
  <div
    className="ui container"
    style={
      Object {
        "paddingTop": "1rem",
      }
    }>
    <div
      className="ui centered relaxed grid"
      style={
        Object {
          "paddingTop": "16rem",
        }
      }>
      <div
        className="center aligned middle aligned sixteen wide column ppd-padding-1"
        style={
          Object {
            "maxWidth": "500px",
          }
        }>
        <div
          className="ui raised segment"
          style={
            Object {
              "paddingTop": "2rem",
            }
          }>
          <img
            alt={undefined}
            className="ui small centered image"
            height={undefined}
            src="logo_sm.png"
            width={undefined} />
          <div
            style={
              Object {
                "padding": "2rem 1rem 0 1rem",
              }
            }>
            <button
              className="ui large button"
              onClick={[Function]}
              style={
                Object {
                  "backgroundStyle": "#d4d4d5",
                }
              }
              tabIndex={undefined}>
              <i
                aria-hidden="true"
                className="google icon" />
               
              Login with Google
            </button>
            <div
              className="ui horizontal divider"
              style={
                Object {
                  "margin": 0,
                  "paddingTop": "1rem",
                }
              }>
              Or
            </div>
          </div>
          <form
            className="ui large form"
            onSubmit={[Function]}>
            <div
              className="ui basic segment">
              <div
                className="field">
                <div
                  className="ui left icon input">
                  <i
                    aria-hidden="true"
                    className="user icon" />
                  <input
                    name="email"
                    onChange={[Function]}
                    placeholder="Email"
                    required={undefined}
                    type="email"
                    value="" />
                </div>
              </div>
              <div
                className="field">
                <div
                  className="ui left icon input">
                  <i
                    aria-hidden="true"
                    className="lock icon" />
                  <input
                    name="password"
                    onChange={[Function]}
                    placeholder="Password"
                    required={undefined}
                    type="password"
                    value="" />
                </div>
              </div>
              <div
                className="field">
                <button
                  className="ui large fluid primary button"
                  onClick={[Function]}
                  required={undefined}
                  style={
                    Object {
                      "backgroundColor": "#1CAF9A",
                    }
                  }
                  tabIndex={undefined}
                  type={undefined}>
                  LOGIN
                </button>
              </div>
            </div>
          </form>
          <div
            className="ui error hidden message">
            <div
              className="content">
              <div
                className="header">
                Login Error
              </div>
              <p>
                Please enter valid credentials.
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
